static/src/stylesheets/module/atoms/_youtube.scss

```diff
@@ -54,6 +54,41 @@
     position: relative;
 }
 
+.youtube-media-atom__paid-for-overlay {
+    background-color: #ffffff;
+    width: 100%;
+    position: absolute;
+    bottom: 0;
+    height: 85px;
+    opacity: .5;
+}
+
+.youtube-media-atom__paid-for-label {
+    font-size: 16px;
+    color: #000000;
+    margin: 5px 10px 0 0;
+    @include mq($until: 590px) {
+        display: none;
+    }
+}
+
+.youtube-media-atom__paid-for-logo-container {
+    display: flex;
+    position: absolute;
+    bottom: 0 ;
+    right: 0;
+    width: auto;
+}
+
+.youtube-media-atom__paid-for-logo {
+    width: auto !important;
```

Comment:
Is it necessary to use `!important` on all these rules as `youtube-media-atom__paid-for-logo` is a new class name? I know `!important` is often used to counteract issues with CSS specificity, but I've found it introduces problems itself where the expected behaviour of specificity is overruled because an `!important` has been used, and the only way to override it is to use another  `!important` and soon they're everywhere in our styles.

Reply:
The image was inheriting styles from a very generic class `u-responsive-ratio` which i wanted to avoid doing the `.not` selector on it. I have now changed the youtube template to conditionally use the `u-responsive-ratio`  class based on `isPaidFor` flag and removed the !importants. Still seems fine and responsive 👍